### ISSUE TITLE
Segmentation fault caused by *_quant_edgetpu.tflite

### ISSUE THREAD

**Issue by victoryred** [google-coral/edgetpu#674]
### Description

I'm using the USB Accelerator.
Tetra@raspberrypi4:~/coral/pycoral $ lsusb
Bus 002 Device 004: ID 090c:1000 Silicon Motion, Inc. - Taiwan (formerly Feiya Technology Corp.) Flash Drive
Bus 002 Device 005: ID 18d1:9302 Google Inc. 
Bus 002 Device 002: ID 05e3:0626 Genesys Logic, Inc. USB3.1 Hub
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 005: ID 046d:c077 Logitech, Inc. M105 Optical Mouse
Bus 001 Device 004: ID 046d:c31c Logitech, Inc. Keyboard K120
Bus 001 Device 003: ID 05e3:0610 Genesys Logic, Inc. Hub
Bus 001 Device 002: ID 2109:3431 VIA Labs, Inc. Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

I've discovered that the classify_image.py yields the following results-
Tetra@raspberrypi4:~/coral/pycoral $ python examples/classify_image.py --model test_data/mobilenet_v2_1.0_224_inat_bird_quant_edgetpu.tflite --labels test_data/inat_bird_labels.txt --input test_data/parrot.jpg
Segmentation fault.....However.......this works.

Tetra@raspberrypi4:~/coral/pycoral $ python examples/classify_image.py --model test_data/mobilenet_v2_1.0_224_inat_bird_quant.tflite --labels test_data/inat_bird_labels.txt --input test_data/parrot.jpg

---INFERENCE TIME----
Note: The first inference on Edge TPU is slow because it includes loading the model into Edge TPU memory.
123.5ms
95.2ms
94.6ms
94.0ms
93.8ms
-------RESULTS--------
Ara macao (Scarlet Macaw): 0.77734

Anybody know about any problems with the edgetpu models running on a Raspberry Pi4 (Bullseye)?

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Bug

### Operating System

Linux

### Coral Device

USB Accelerator

### Other Devices

Rapsberry Pi 4

### Programming Language

Python 3.9

### Relevant Log Output

_No response_</details>

**Comment by hjonnala** [comment]
Hi @victoryred could you please if you have the same setup as mentioned at: https://github.com/google-coral/edgetpu/issues/660#issuecomment-1249936643.

Can you try the demo with user as  `pi` instead of `Tetra`.

**Comment by victoryred** [comment]
Tetra@raspberrypi4:~ $ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
Tetra@raspberrypi4:~ $ uname -a
Linux raspberrypi4 5.15.61-v8+ #1579 SMP PREEMPT Fri Aug 26 11:16:44 BST 2022 aarch64 GNU/Linux
Tetra@raspberrypi4:~ $ python3
Python 3.9.2 (default, Feb 28 2021, 17:03:44)
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tflite_runtime as tflite
>>> tflite.__version__
'2.10.0'
>>> import pycoral
>>> pycoral.__version__
'2.0.0'
>>>

**Comment by victoryred** [comment]
I have several updates and comments......First of all, thank you for your support.

Secondly, I found several installations of the tflite_runtime installed. I must have become out of sync with all the installs/uninstalls. Again, I was literally following the install instructions on the website. I was tired.....I repeatly uninstalled versions until nothing was found. That was subtle.

Thirdly, looking at the classify_image.py command line it stuck me as 'funny' that the cpu version works under tflite_runtime 2.10 but the edgetpu versions didn't. Will the tflite_runtime get updated and rolled to stay up with the othe python versions. Version 2.5 seems 'old' compared to others.

Finally, unfortunately what got this frenzy started was the problem with apt-keys in the following command-

curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -

apt-key is deprecated and when I had problems with this command I was launched looking for something that works. Can someone please scrub/clean/simplify the usb accelerator installation instructions webpage and fix the
 curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -

so the line doesn't fail? Thank you everyone. I hope I've made it easier for the next guy.

**Comment by hjonnala** [comment]
> apt-key is deprecated and when I had problems with this command I was launched looking for something that works. Can someone please scrub/clean/simplify the usb accelerator installation instructions webpage and fix the
> curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -

Not sure, what error you are receiving but this line should work normally..
![image](https://user-images.githubusercontent.com/86132058/199648055-049acf6c-82c0-41f9-9856-87d1f69636b3.png)


> Thirdly, looking at the classify_image.py command line it stuck me as 'funny' that the cpu version works under tflite_runtime 2.10 but the edgetpu versions didn't. Will the tflite_runtime get updated and rolled to stay up with the othe python versions. Version 2.5 seems 'old' compared to others.

Unfortuanltey, we don't have nay planned updates to update the tflite_runtime and pycoral versions.

**Comment by vilmos-balazs** [comment]
Just in case it helps somebody...
I was struggling with this error on this Raspbian: 
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

an I found  this Notes:
 Note: We no longer update the Debian package python3-tflite-runtime. The latest Debian package is for TF version 2.5, which you can install by following [these older instructions](https://github.com/tensorflow/tensorflow/blob/v2.5.0/tensorflow/lite/g3doc/guide/python.md#install-tensorflow-lite-for-python).

So I had to uninstall the latest version at this time (2.10):
   $  pip uninstall tflite-runtime
And install the old 2.5 version:
   $  sudo apt-get install python3-tflite-runtime

After this I was able to run the bird classifier on CORAL accelerator.

**Comment by Dionysus326** [comment]
> Just in case it helps somebody... I was struggling with this error on this Raspbian: PRETTY_NAME="Debian GNU/Linux 11 (bullseye)" NAME="Debian GNU/Linux" VERSION_ID="11" VERSION="11 (bullseye)" VERSION_CODENAME=bullseye ID=debian HOME_URL="https://www.debian.org/" SUPPORT_URL="https://www.debian.org/support" BUG_REPORT_URL="https://bugs.debian.org/"
> 
> an I found this Notes: Note: We no longer update the Debian package python3-tflite-runtime. The latest Debian package is for TF version 2.5, which you can install by following [these older instructions](https://github.com/tensorflow/tensorflow/blob/v2.5.0/tensorflow/lite/g3doc/guide/python.md#install-tensorflow-lite-for-python).
> 
> So I had to uninstall the latest version at this time (2.10): $ pip uninstall tflite-runtime And install the old 2.5 version: $ sudo apt-get install python3-tflite-runtime
> 
> After this I was able to run the bird classifier on CORAL accelerator.

Thank you. I encountered the same issue and solved by uninstalling tflite-runtime using pip
